src/main/java/tasklist/Duke.java

```diff
@@ -0,0 +1,49 @@
+package tasklist;
+
+import java.util.Scanner;
+
+import tasklist.commands.Command;
+
+/** Main class of the chatbot application. Initializes the application */
+public class Duke {
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    /** constructor implementation */
+    public Duke() {
+        ui = new Ui();
+        storage = new Storage("./data/duke.ser");
+        taskList = storage.loadTasks();
+        System.out.println(taskList.getTask(0));
+    }
+
+    // /** Function to start running the chatbot */
```

Comment:
I noticed there are sections of code that are currently commented out and appear to be unused. It might be beneficial to remove these to keep the codebase clean and maintainable. If there's a specific reason they're being kept, could you please clarify?